src/components/DayPicker.jsx

```diff
@@ -52,6 +54,8 @@ const defaultProps = {
   modifiers: {},
   orientation: HORIZONTAL_ORIENTATION,
   withPortal: false,
+  focused: true,
```

Comment:
i agree that defaulting to true is confusing, but in this case it might make sense. It seems unintuitive for people to have to specify `focused` when rendering the DayPicker, the default behavior should be "if you render this component, it will show".

I could change the name, but then it will deviate from the prop names used in `SDP` and `DRP`... if you think that it would still be best i can make it `hidden`?